apisix/init.lua

```diff
@@ -629,6 +630,13 @@ end
 function _M.http_header_filter_phase()
     core.response.set_header("Server", ver_header)
 
+    local up_status = get_var("upstream_status")
+    local from, _ = re_find(up_status, "5[0-9]{2}$", "jo")
```

Comment:
string.sub (upstatus,  -3)  

Should be simpler

Reply:
Yes i will update later.

Reply:
done.